config-bootstrapper/src/main/resources/configs/config-bootstrapper/attribute-service/hypertrace/hypertrace_api_trace_attributes.conf

```diff
@@ -217,6 +217,15 @@ commands = [
               scope: API_TRACE,
               sources: [QS],
               type: ATTRIBUTE
+            },
+            {
+              fqn: Api.Trace.api_callee_name_count,
+              key: apiCalleeInfo,
```

Comment:
Should we keep key as `apiCalleeNameCount`? It will align with <scope>.<key> format of fqn

Reply:
done